core/src/main/java/org/apache/carbondata/core/util/DataTypeUtil.java

```diff
@@ -401,6 +401,11 @@ public static Object getDataDataTypeForNoDictionaryColumn(String dimensionValue,
     } else if (actualDataType == DataTypes.LONG) {
       return ByteUtil.toBytes((Long) dimensionValue);
     } else if (actualDataType == DataTypes.TIMESTAMP) {
+      if (dimensionValue instanceof String) {
```

Comment:
Avro supports timestamp as logical type. So need not support this, as avro supports specific data type

Reply:
This fix is removed from this PR as it is not applicable